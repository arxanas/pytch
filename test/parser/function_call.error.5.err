EXPECTED_RPAREN[1012] in test/parser/function_call.error.5.pytch, line 7, character 15:
Error: I was expecting a ')' to indicate the end of this function argument
list, but instead got the end of the file.
   +--------------------------------------------------------------------------+
   | test/parser/function_call.error.5.pytch                                  |
 1 | print(                                                                   |
   |      ^ Note: The beginning of the argument list is here.                 |
 2 |   1,                                                                     |
   +--------------------------------------------------------------------------+
 6 |           5,                                                             |
 7 |             6,                                                           |
   |               ^ Error: I was expecting a ')' to indicate the end of this |
   | function argument list, but instead got the end of the file.             |
   +--------------------------------------------------------------------------+
