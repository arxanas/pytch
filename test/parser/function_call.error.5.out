SyntaxTree
    FunctionCallExpr
        IdentifierExpr
            Token 'print'
        ArgumentList
            Token '('
            Trailing '\n'
            Argument
                IntLiteralExpr
                    Leading '  '
                    Token '1'
                Token ','
                Trailing '\n'
            Argument
                IntLiteralExpr
                    Leading '    '
                    Token '2'
                Token ','
                Trailing '\n'
            Argument
                IntLiteralExpr
                    Leading '      '
                    Token '3'
                Token ','
                Trailing '\n'
            Argument
                IntLiteralExpr
                    Leading '        '
                    Token '4'
                Token ','
                Trailing '\n'
            Argument
                IntLiteralExpr
                    Leading '          '
                    Token '5'
                Token ','
                Trailing '\n'
            Argument
                IntLiteralExpr
                    Leading '            '
                    Token '6'
                Token ','
                Trailing '\n'
            <missing>
    Token EOF ''
